rpms.yaml

```diff
@@ -0,0 +1,31 @@
+# brew package names
+rpms:
+# rhcos dependencies
+- butane
+- clevis
+- console-login-helper-messages
+- coreos-installer
+- ignition
+- nss-altfiles
+- ostree
+- podman
+- redhat-release-coreos
+- rhcos-tools
+- rust-afterburn
+- rust-bootupd
+- skopeo
+- toolbox
+- conmon
+- kata-containers
+# that we build
+- openshift-ansible
+- openshift-clients
+- openshift-enterprise-service-idler
+- openshift-kuryr
+- openshift
+# sometimes we ship these
+# when critical fixes arrive
```

Comment:
likewise skopeo and podman

if rhel has shipped a newer one, we don't need to

Reply:
added